owncloudApp/src/main/java/com/owncloud/android/presentation/logging/RecyclerViewLogsAdapter.kt

```diff
@@ -84,4 +87,11 @@ class RecyclerViewLogsAdapter(
         fun open(file: File)
         fun download(file: File)
     }
+
+
+    private fun hideDownloadButton(holder: ViewHolder) {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.Q) {
+            holder.binding.imageViewDownloadActivityLogsList.isVisible = false
+        }
```

Comment:
I would add these 3 lines directly in the `onBindViewHolder` method, having a method called `hide...` that is always called can lead to confusion, and we can save calling to `holder.binding` again

Reply:
Done